config_defaults.yml

```diff
@@ -41,3 +42,5 @@ Rakefile:
   - 'disable_class_inherits_from_params_class'
   - 'disable_documentation'
   - 'disable_single_quote_string_with_variables'
+spec/spec.opts:
+  delete: true
```

Comment:
I haven't ever been able to make the delete function work and haven't gotten around to troubleshooting it.